src/pages/main/collaboration/stream/stream.ts

```diff
@@ -58,17 +58,14 @@ export const useCollabStream = (dispatch: (action: CollabReducerAction) => void)
   );
 
   const connectionStateRef = useRef<ReadyState>(connectionState);
+  connectionStateRef.current = connectionState;
 
   const sendMessage = (message: Message) => {
     if (connectionStateRef.current === ReadyState.OPEN) {
       send(JSON.stringify(message));
     }
   };
 
-  useEffect(() => {
-    lastMessage && handleIncomingMessage(lastMessage.data);
-  }, [lastMessage]);
-
```

Comment:
Is this replaced already somewhere?

Reply:
https://github.com/Agar-OSS/agartex-app/blob/monaco-goes-collaboration-fix/src/pages/main/collaboration/stream/stream.ts#L56